AES.Web/Views/Application/Education.cshtml

```diff
@@ -0,0 +1,117 @@
+                    </div>
+                    @* Zip *@
+                    @Html.LabelFor(m => m.Zip, new { @class = "col-md-2 control-label" })
+                    <div class="col-md-2">
+                        @Html.TextBoxFor(m => m.Zip, new { @class = "form-control" })
+                        @Html.ValidationMessageFor(m => m.Zip, "", new { @class = "text-danger" })
+                    </div>
+                </div>
+
+                <div class="form-group">
+                    @* Start Date *@
+                    @Html.LabelFor(m => m.StartDate, new { @class = "col-md-2 control-label" })
+                    <div class="col-md-2">
+                        @Html.TextBoxFor(m => m.StartDate, new { @class = "form-control" })
+                        @Html.ValidationMessageFor(m => m.StartDate, "", new { @class = "text-danger" })
+                    </div>
+
+                    @* End Date *@
+                    @Html.LabelFor(m => m.EndDate, new { @class = "col-md-2 control-label" })
+                    <div class="col-md-2">
+                        @Html.TextBoxFor(m => m.EndDate, new { @class = "form-control" })
+                        @Html.ValidationMessageFor(m => m.EndDate, "", new { @class = "text-danger" })
+                    </div>
+                </div>
+
+                <div class="form-group">
+                    @* Degree *@
+                    @Html.LabelFor(m => m.Degree, new { @class = "col-md-2 control-label" })
+                    <div class="col-md-3">
+                        @Html.TextBoxFor(m => m.Degree, new { @class = "form-control" })
+                        @Html.ValidationMessageFor(m => m.Degree, "", new { @class = "text-danger" })
+                    </div>
+                </div>
+
+                <div class="form-group">
+                    @* Major *@
+                    @Html.LabelFor(m => m.Major, new { @class = "col-md-2 control-label" })
+                    <div class="col-md-3">
+                        @Html.TextBoxFor(m => m.Major, new { @class = "form-control" })
+                        @Html.ValidationMessageFor(m => m.Major, "", new { @class = "text-danger" })
+                    </div>
+                </div>
+
+                <div class="form-group">
+                    @* Graduation Date *@
+                    @Html.LabelFor(m => m.GraduationDate, new { @class = "col-md-2 control-label" })
+                    <div class="col-md-2">
+                        @Html.TextBoxFor(m => m.GraduationDate, new { @class = "form-control" })
+                        @Html.ValidationMessageFor(m => m.GraduationDate, "", new { @class = "text-danger" })
+                    </div>
+                </div>
+
+                <div class="form-group">
+                    <div class="bottom-left">
+                        <input type="submit" value="Proceed to References" class="btn btn-default" />
```

Comment:
Need a "Back" button too. Ditto on all pages.

A nicer way to do this would be to have the buttons look like arrows, and then just have the section they are going to. "Next" and "Previous" would also suffice.